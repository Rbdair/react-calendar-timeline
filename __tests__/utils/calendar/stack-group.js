import { stackGroup } from 'lib/utility/calendar'
import { dimensionItems } from '../../../__fixtures__/groupOrderAndItemDimentions'

describe('stackGroup', ()=>{
    it('should stack list of items space', ()=>{
        expect(stackGroup(dimensionItems, 'space', 30, 0)).toMatchSnapshot()
    })
    it('should stack list of items lines', ()=>{
        expect(stackGroup(dimensionItems, 'lines', 30, 0)).toMatchSnapshot()
    })
    it('should not stack list of items', ()=>{
        expect(stackGroup(dimensionItems, false, 30, 0)).toMatchSnapshot()
    })
})