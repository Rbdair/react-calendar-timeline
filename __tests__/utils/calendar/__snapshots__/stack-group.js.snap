// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stackGroup should not stack list of items 1`] = `
Object {
  "groupHeight": 0,
  "verticalMargin": 0,
}
`;

exports[`stackGroup should stack list of items 1`] = `
Object {
  "groupHeight": 0,
  "verticalMargin": 3.75,
}
`;

exports[`stackGroup should stack list of items lines 1`] = `
Object {
  "groupHeight": 180,
  "verticalMargin": 7.5,
}
`;

exports[`stackGroup should stack list of items space 1`] = `
Object {
  "groupHeight": 0,
  "verticalMargin": 7.5,
}
`;
